commands/command_pictures.cpp

```diff
@@ -24,7 +24,7 @@ void SetPictureOffset::redo()
 {
 	picture *pic = dive_get_picture(d, filename);
 	if (!pic) {
-		fprintf(stderr, "SetPictureOffset::redo(): picture disappeared!");
+		fprintf(stderr, "SetPictureOffset::redo(): picture disappeared!\n");
```

Comment:
my understanding was that `stderr` wasn't line buffered. This doesn't appear to be part of any standard, that seems to say it is implementation dependent, but I saw several references to this as "the convention" and lots of code appears to assume that this is indeed the case.
All that said, the new line makes this much more readable.